storage/StorageInterface.h

```diff
@@ -214,6 +214,23 @@ namespace storage
 
 
     /**
+     * Abstract class for some callbacks during commit. Currently only works
+     * for btrfs as root filesystem.
+     */
+    class CommitCallbacks
+    {
+    public:
+
+	virtual ~CommitCallbacks() {}
+
+	virtual void post_root_filesystem_create() const {}
+	virtual void post_root_mount() const {}
+	virtual void post_root_fstab_add() const {}
```

Comment:
I understand both ways.

What is actually wrong with my code? Is this nitpicking and prevents a LGTM?